### PR TITLE
javadoc

### PR DESCRIPTION
**Pull request by ViktoriaVass**
documentation

### CODE REVIEW

**Comment by muhlba91** [comment]
this is not javadoc - you can find an example here: https://github.com/muhlba91/fh-burgenland-bswe-3b/blob/sample/src/main/java/io/muehlbachler/bswe3b/CalculatorService.java

your title and commit message should  be more clear, like `doc: add javadoc for divide`

File: src/main/java/io/muehlbachler/bswe3b/CalculatorService.java [hunk 1]
```diff
@@ -5,9 +5,14 @@
 // calculator - documentation n
 @Service
 public class CalculatorService {
-    // divide - should return a/b
-    public int divide(final int a, final int b) {
-        // FIXME: implement
-        return -1;
+   public int divide(final int a, final int b) {
+        if (b == 0) {
```
Comment (muhlba91) [inline src/main/java/io/muehlbachler/bswe3b/CalculatorService.java:9]: the spec was if b <= 0, return 0.

so this would not adhere to those specs BUT if you want to throw an exception you should test for that case as well. you can use `assertThrows` for that: https://junit.org/junit5/docs/5.8.2/api/org.junit.jupiter.api/org/junit/jupiter/api/Assertions.html#assertThrows(java.lang.Class,org.junit.jupiter.api.function.Executable).

File: src/test/java/io/muehlbachler/bswe3b/CalculatorServiceTest.java [hunk 2]
```diff
@@ -11,5 +11,13 @@ public void testDividePositiveNumberNoRemainder() {
         assertEquals(1, calc.divide(1, 1), "1/1 should be 1");
     }
 
-    // FIXME: implement more tests
+    @Test
```
Comment (muhlba91) [inline src/test/java/io/muehlbachler/bswe3b/CalculatorServiceTest.java:14]: i'd expect more test case, especially around edge cases.
examples: b = 0, b < 0.
